README.md

```diff
@@ -93,6 +104,16 @@ if err != nil {
 }
 ```
 
+### GetAccessToken()
+
+`func (sf *Salesforce) GetAccessToken() string`
```

Comment:
````suggestion
```go
func (sf *Salesforce) GetAccessToken() string
```
````
This way the code will be formatted with Go syntax

Reply:
@ccoVeille I was specifically not formatting it that way for the headers here, but its a good thing to note. Thank you.